docs/src/user-guide/reference-text-search-syntax.md

```diff
@@ -0,0 +1,58 @@
+# Text-log search syntax
+
+To search text logs, CLP currently supports wildcard queries. A wildcard query is a query where:
+
+* `*` matches zero or more characters
+* `?` matches any single character
+
+To search for a wildcard literally, you can escape it using a backslash (`\`). To search for a
+backslash literally, you can escape it with another backslash.
+
+By default, CLP treats queries as substring searches (alike `grep`). So the query
+`container * failed` is interpreted internally as `*container * failed*` (with prefix and suffix
+`*`).
+
+:::{tip}
+Including delimiters (e.g. spaces, colons, etc.) at the beginning and end of your query can improve
+CLP's search performance.
+
+If your query begins/ends with a token, since CLP implicitly adds wildcards to each end of the
```

Comment:
I am not sure if the definition of "token" is clear to user (disclaimer: I am not a English native speaker). Do we need to say something like (i.e. non-delimiter character)?